src/main/java/core/basesyntax/model/figure/Circle.java

```diff
@@ -1,13 +1,44 @@
 package core.basesyntax.model.figure;
 
+import java.util.Objects;
+import javax.persistence.Entity;
+
+@Entity
 public class Circle extends Figure {
     private int radius;
 
+    public Circle() {
+    }
```

Comment:
Empty constructor is created by default so we don't need it if we don't have another constructors with parameters. But it is not a mistake I think.

Reply:
it is better be removed if it's redundant

Reply:
Fixed